tensorflow_quantum/core/serialize/serializer.py

```diff
@@ -26,6 +27,10 @@
 _CONSTANT_TRUE = lambda x: True
 
 
+def _round(x):
+    return np.round(x, 6) if isinstance(x, float) else x
```

Comment:
Are we ok with the fact that:
```python3
isinstance(np.float32(0.1), float) => False
```
 ?

Reply:
hmm we don't use `np.float32`. if we use, even if the value itself is exactly the same, circuit equality failed due to types.

Reply:
so it is intended.

Reply:
The reason why I failed to update serializer was using `np.float32`. I realized that `np.round` itself doesn't require the data type, so I could do it today :)